py-polodb/src/py_database.rs

```diff
@@ -0,0 +1,146 @@
+
+use polodb_core::bson::Document;
+use polodb_core::{Collection, CollectionT, Database};
+use pyo3::exceptions::PyOSError;
+use pyo3::exceptions::PyRuntimeError; // Import PyRuntimeError for error handling
+use pyo3::prelude::*;
+use pyo3::types::PyDict;
+use std::borrow::Borrow;
+use std::path::Path;
+use std::sync::{Arc, Mutex};
+
+use crate::doc_bson_to_py_translator::{bson_to_py_obj,convert_py_obj_to_document,document_to_pydict};
+
+
+
```

Comment:
It seems there are too many spaces here. Can we leave only 1-2 empty lines here?

Reply:
sûre, reformatted